wandb/apis/public.py

```diff
@@ -822,17 +822,16 @@ def __init__(self, client, entity, project, run_id, attrs={}):
         self._entity = entity
         self.project = project
         self._files = {}
-        self._base_dir = env.get_dir(tempfile.gettempdir())
+        self._base_dir = tempfile.gettempdir()
```

Comment:
Again, I would think we want to respect a user specified directory if they set it.